src/covid19model/models/base.py

```diff
@@ -27,12 +27,13 @@ class BaseModel:
     parameter_names = None
     parameters_stratified_names = None
     stratification = None
-    solver = None
+    apply_compliance_to = None
```

Comment:
@jorisvandenbossche Compliance is indeed a "niche" concept, only to be applied to the parameter governing social interaction. In our models, this is `Nc`, but if we were to define a SIR model with two parameters,
> - `beta`: rate of spread = social interaction + infectiousness upon contact
> - `gamma`: recovery rate
The user would want to apply compliance to `beta`. The purpose of having the user define an `apply_compliance_to = paramater_name` is so that it can be applied to any parameter governing social interaction.